packages/components/src/RichTooltip/RichLayout/RichLayout.component.js

```diff
@@ -5,7 +5,7 @@ import Inject from '../../Inject';
 import theme from './RichLayout.scss';
 
 const TooltipPropTypes = {
-	id: PropTypes.string.isRequired,
+	id: PropTypes.string,
```

Comment:
So you will have `undefined-content` as id sometimes `id={`${id}-content`}`

Reply:
For Body yes but Header and Footer no
And using class undefined-content is dirty. Id should be uniq